validator/validator.protoascii

```diff
@@ -4050,6 +4050,25 @@ tags: {  # <amp-install-serviceworker>
     supported_layouts: NODISPLAY
   }
 }
+tags: {  # <amp-jwplayer>
```

Comment:
Let's remove this from here for now and send post this fragment into a separate issue for validator. It might be faster for validator folk to update this in batch with other pushes they do.